File: services/telegramservice/DTOs/DTO/WordGroupAllDTO.cs
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;
using System.Linq;
using System.Threading.Tasks;

namespace DTOs.DTO
{
    public class WordGroupAllDTO
    {
        public long? Id { get; set; }

        public long? IdGroup { get; set; }

        public string? InfoGroup { get; set; }

        public string? TitleGroup { get; set; }

        public string? FindGroup { get; set; }

        public string? HashGroup { get; set; }

        public long? IdUserJoin { get; set; }

        public int? Type { get; set; }

        public long? HandlersId { get; set; }

        public DateTime? LastUpdate { get; set; }

        public DateTime? LastHandle { get; set; }

        public int? TotalSendRequest { get; set; } = 0;

        public int? TotalDetectPrivate { get; set; } = 0;

        public long? LinkedId { get; set; }
        public long? ParticipantsCount { get; set; }
        public DateTime? CreatedDate { get; set; }
        public string? Flags { get; set; }
        public string? Flags2 { get; set; }
    }
}

File: services/dbservice/src/main/java/com/torin/dbService/dto/WordGroupAllDto.java
package com.torin.dbService.dto;

import java.time.Instant;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@NoArgsConstructor
@AllArgsConstructor
public class WordGroupAllDto {

    private Long id;

    private Long idGroup;

    private String infoGroup;

    private String titleGroup;

    private String findGroup;

    private String hashGroup;

    private Long idUserJoin;

    private Integer type;

    private Long handlersId;

    private Instant lastUpdate;

    private Instant lastHandle;

    private Integer totalSendRequest = 0;

    private Integer totalDetectPrivate = 0;

    private Long linkedId;

    private Long participantsCount;

    private Instant createdDate;
    
    private String flags;

    private String flags2;
}
